newsletter/admin.py

```diff
@@ -43,13 +44,22 @@
 
 from .settings import newsletter_settings
 
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
I moved it because `flake8` (and PEP 8) complains if code is mixed into the imports. I already moved most of them in 485f99293c280fd6ac99af653736a04804f794d7 but I had this branch in-progress so I didn't move it then to avoid a merge conflict.

IMO it's a good idea to keep close to the PEP8 style, so I run `flake8` to find and clean up code that doesn't match it. Once the code base doesn't have any non-PEP8 code we can add a `flake8` build like [django-extensions has](https://github.com/django-extensions/django-extensions/blob/master/.travis.yml) to make sure the coding style stays clean.